conf/nginx.conf

```diff
@@ -14,6 +14,13 @@
 # Default server configuration
```

Comment:
I think we should move this file to conf/nginx/nginx.conf (which currently seems to contain the default nginx.conf)

Reply:
ah I'm confused, this nginx.conf file is only for docker, which does not use the files in sites-available?

Reply:
yes for docker we have a different config than in prod.

So I kept the nginx conf for prod in conf/nginx, while docker simply uses conf/nginx.conf
(a docker nginx instance only serve one application, and obf, opf, etc. are not docker ready yet.)